service/version_bundles.go

```diff
@@ -32,6 +33,7 @@ func NewVersionBundles() []versionbundle.Bundle {
 	versionBundles = append(versionBundles, v12patch1.VersionBundle())
 	versionBundles = append(versionBundles, v13.VersionBundle())
 	versionBundles = append(versionBundles, v14.VersionBundle())
+	versionBundles = append(versionBundles, v15.VersionBundle())
```

Comment:
The release wasn't showing on ginger because the version bundle wasn't wired. Getting that sorted.

cc @pipo02mix

Reply:
@corest that also explains why we didn't see it.